const makeError = () => {
  try {
    const name = "Ernest";
    name = Yoyowah;
  } catch (err) {
    console.log(err);
  }
};

makeError();
